src/tests/panel.test.ts

```diff
@@ -0,0 +1,208 @@
+import {
+    _getPanelController, _getPanelsFromPanelSetIDController, _getPanelBasedOnPanelSetAndIndexController, _createPanelController
+} from '../requestHandlers/panel';
+
+import * as panelService from '../services/panelService';
+import { getPanelSetByID } from '../services/panelSetService';
+import { Sequelize } from 'sequelize';
+jest.mock('../services/panelService');
+jest.mock('../services/panelSetService');
+
+let sequelizeMock: jest.Mocked<Sequelize>;
+describe('_getPanelBasedOnPanelSetIDAndIndex', () => {
+    beforeEach(() => {
+        sequelizeMock = {} as jest.Mocked<Sequelize>;
+    });
```

Comment:
This `beforeEach` can be moved outside of the describe. This means other `beforeEach` can be removed